docs/workload.rst

```diff
@@ -0,0 +1,193 @@
+VM Workloads Search
+==============
+
+Visualize the inventory of vSphere workloads that do not have Carbon Black Cloud sensors installed.
+
+Fetch Compute Resource by ID
+----------------------------
+
+Using a query of the ``ComputeResource`` object, you can get the compute resource by ID from your organization.
```

Comment:
Can we add a text blurb here indicating that a compute resource is a vm without a sensor installed